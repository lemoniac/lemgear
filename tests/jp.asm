jp a

b:
halt

a:
	ld A, 10
	jp b
