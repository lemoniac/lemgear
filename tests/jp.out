AF : 0A00
